asyncapi/asyncapi.yml

```diff
@@ -363,6 +363,36 @@ channels:
           mqtt:
             qos: 1
             retain: false
+  pe/dpi/journey_status:
```

Comment:
Perhaps we could combine these two topics and produce the journey_status message both on new journey and periodically every 5 min?

Reply:
Yes, we should do it like that.